src/remote_common.c

```diff
@@ -64,15 +64,16 @@ bool die_on_error(int x, char const *context)
     return true;
 }
 
-bool connect_broker(amqp_connection_state_t *conn)
+bool connect_broker(amqp_connection_state_t *conn, const char *host)
 {
     amqp_socket_t *socket = NULL;
+    const char *hostIP = (host != NULL) ? host : "localhost";
```

Comment:
Change from `hostIP` to `hostname`.

Reply:
Alternatively, using simple `host` works as well.

Reply:
I change the local variable to `host` and the input parameter of `connect_broker()` to `hostName`.
Since the original parameter naming of `amqp_socket_open()` is `host`.